module/config.mjs

```diff
@@ -1669,8 +1669,18 @@ DND5E.encumbrance = {
     }
   },
   speedReduction: {
-    encumbered: 10,
-    heavilyEncumbered: 20
+    encumbered: {
+      ft: 10,
+      m: 3
+    },
+    heavilyEncumbered: {
+      ft: 20,
+      m: 6
+    },
+    exceedingCarryingCapacity: {
+      ft: 5,
+      m: 1.5
+    }
```

Comment:
Perhaps we should have a `CONFIG.DND5E.feetToMeters` conversion factor instead and have only values in feet in your config.